### DISCUSSION TITLE
How to set scrollbar init scrollTo bottom scroll?

### DISCUSSION THREAD

**Discussion by dropways** [idiotWu/smooth-scrollbar#324] [Q&A] [answered]
I want to use in chat area

**Comment by dolphin-wood** [comment, marked as answer]
Try:

```js
scrollbar.scrollTop = scrollbar.limit.y
```

**Reply by Illia-Linevych** [reply]
Is there any way to do the same but with smooth scrolling?

**Reply by dolphin-wood** [reply]
@Chameleon45 You can use the [`.scrollTo()`](https://github.com/idiotWu/smooth-scrollbar/blob/develop/docs/api.md#scrollbarscrollto) method or the [`.setMomentum()`](https://github.com/idiotWu/smooth-scrollbar/blob/develop/docs/api.md#scrollbarsetmomentum) method.

**Reply by Illia-Linevych** [reply]
Oh, I found it too and it is working as expected. Thank you)
